app/controllers/users_controller.rb

```diff
@@ -71,6 +78,15 @@ def list
     @letters = ('A'..'Z').to_a
   end
 
+  def review
```

Comment:
Any suggestions on where to move this method. Should it be under requested_users_controller.rb?